appservice/src/connectToGitHub.ts

```diff
@@ -108,53 +106,40 @@ export async function connectToGitHub(node: AzureTreeItem, client: SiteClient):
     }
 }
 
-async function showGitHubAuthPrompt(): Promise<void> {
-    const learnMore: string = localize('learnMore', 'Learn More');
-    const setupGithub: string = localize('setupGithub', 'You must give Azure access to your GitHub account.');
-    const input: string | undefined = await vscode.window.showErrorMessage(setupGithub, learnMore);
-    if (input === learnMore) {
-        // tslint:disable-next-line:no-unsafe-any
-        opn('https://aka.ms/B7g6sw');
+async function showGitHubAuthPrompt(ti: AzureTreeItem<ISiteTreeRoot>, error: Error): Promise<void> {
+    const invalidToken: string = localize('tokenExpired', 'Azure\'s GitHub token in invalid.  Authorize in the "Deployment Center"');
+    const goToPortal: string = localize('goToPortal', 'Go to Portal "Deployment Center"');
```

Comment:
Can I get a screenshot of this? Seems like a long button title...

Reply:
Yeah, it does a bit awkward to me to.  Maybe just "Go to Portal"?
![image](https://user-images.githubusercontent.com/5290572/48233320-98d57f80-e369-11e8-8aa7-0782f526faea.png)